pom.xml

```diff
@@ -264,6 +264,7 @@
     <spark.test.home>${session.executionRootDirectory}</spark.test.home>
     <spark.test.webdriver.chrome.driver></spark.test.webdriver.chrome.driver>
     <spark.test.docker.keepContainer>false</spark.test.docker.keepContainer>
+    <spark.test.docker.removePulledImage>false</spark.test.docker.removePulledImage>
```

Comment:
I think we should maybe just enable it by default. but I will leave it to you @sarutak and @dongjoon-hyun

Reply:
I disabled it by default just for behavior compatibility (pulled images are not removed before this change).
But it may be no problem even if we enable it.